tests/models/test_modeling_common.py

```diff
@@ -1941,7 +1941,7 @@ def test_passing_dict_device_map_works(self, name, device):
             _ = loaded_model(**inputs_dict)
 
 
-class VAETestMixin:
+class AutoencoderTesterMixin:
```

Comment:
Since we're probably going to add quite a few Mixins during refactors for tests, let's move this model specific ones into the the model test modules themselves. So we would add a `testing_utils.py` under `autoencoders` and this Mixin would go in there.